htlcswitch/link.go

```diff
@@ -870,6 +874,11 @@ func (l *channelLink) htlcManager() {
 	log.Infof("HTLC manager for ChannelPoint(%v) started, "+
 		"bandwidth=%v", l.channel.ChannelPoint(), l.Bandwidth())
 
+	// Funding locked has already been received, so we'll go ahead and
+	// deliver the active channel notification since EligibleToForward
+	// returns true now that the link has been added to the switch.
+	l.cfg.NotifyActiveChannel(*l.ChannelPoint())
```

Comment:
I don't see how this changes anything. Isn't this more or less equivalent to this being called after `AddLink` has been called?